app/views/tags/display.html

```diff
@@ -7,7 +7,15 @@ <h2 class="post-title">
     <div class="post-metadata">
         <span class="post-author">by ${_post.author.fullname}</span>,
         <span class="post-date">${_post.postedAt.format('dd MMM yy')}</span>
+
         #{if _as != 'full'}
+            <span class="post-likes">
+                &nbsp;|&nbsp; ${_post.likes.size() ?: 'no'}
```

Comment:
generally we want to use CSS spacing instead of a `&nbsp;` but sometimes it's needed. I could see arguments both ways for this case